pkg/retistruct/R/server-handlers.R

```diff
@@ -383,6 +383,25 @@ h.mark.od <- function(state, input, output, session, x, y, ...) {
   do.plot(state=state, input=input, output=output)
 }
 
+h.demo1 <- function(state, input, output, session, extdata, directory1, directory2) {
+    state$dataset <- file.path(extdata, directory1, directory2)
+    print(state$dataset)
+    h.open(state, input, output, session)
+}
+
+h.demo2 <- function(state, input, output, session, extdata.demos, directory1, directory2) {
+  dataset <- file.path(extdata.demos, directory1, directory2)
+  print(dataset)
+  if (!file.exists(dataset)) {
+    showNotification(
+      "Install the retistructdemos package using
+      devtools::install_github(\"davidcsterratt/retistruct/pkg/retistructdemos",
```

Comment:
When I copy-and-pasted from this message, I got `devtools::install_github("davidcsterratt/retistruct/pkg/retistructdemos`.

We need to add \") at the end of the string.

I think that instead of "using" we should say "by pasting into the R command prompt the following:"